acrobat/blocks/prompt-card/prompt-card.css

```diff
@@ -0,0 +1,158 @@
+#prompt {
+  display: none
+}
+
+.prompt-toast {
+  font-family: "Adobe Clean", adobe-clean, "Trebuchet MS", sans-serif;
+  align-items: flex-start;
+  background: #05834e;
+  border-radius: 10px;
+  color: #fff;
+  display: none;
+  gap: 10px;
+  left: 50%;
+  padding: 20px 16px;
+  position: fixed;
+  top: 110px;
+  transform: translate(-50%, -50%);
+  z-index: 9999
+}
+
+.prompt-toast--show {
+  display: inline-flex
+}
+
+.prompt-toast:before {
+  background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAiIGhlaWdodD0iMjAiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGcgZmlsbD0iI2ZmZiI+PHBhdGggZD0iTTEwIDE4Ljc1Yy00LjgyNSAwLTguNzUtMy45MjUtOC43NS04Ljc1UzUuMTc1IDEuMjUgMTAgMS4yNXM4Ljc1IDMuOTI1IDguNzUgOC43NS0zLjkyNSA4Ljc1LTguNzUgOC43NVptMC0xNkE3LjI1OCA3LjI1OCAwIDAgMCAyLjc1IDEwYzAgMy45OTggMy4yNTIgNy4yNSA3LjI1IDcuMjVzNy4yNS0zLjI1MiA3LjI1LTcuMjVTMTMuOTk4IDIuNzUgMTAgMi43NVoiLz48cGF0aCBkPSJNOS4xODMgMTMuOTQzYS43NTEuNzUxIDAgMCAxLS41NTgtLjI0OGwtMi42NDMtMi45NGEuNzUuNzUgMCAwIDEgMS4xMTUtMS4wMDNsMi4wMjYgMi4yNTQgMy43LTUuMDQ2YS43NS43NSAwIDEgMSAxLjIwOS44ODdsLTQuMjQ1IDUuNzlhLjc0NS43NDUgMCAwIDEtLjU2Ny4zMDVsLS4wMzcuMDAxWiIvPjwvZz48L3N2Zz4=");
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: "";
+  height: 20px;
+  margin-top: 3px;
+  min-width: 20px;
+  width: 20px
+}
+
+.prompt-close {
+  background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTIiIGhlaWdodD0iMTIiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGcgY2xpcC1wYXRoPSJ1cmwoI2EpIj48cGF0aCBkPSJtNy4zNDIgNiAzLjM5Ni0zLjM5NmEuOTUuOTUgMCAwIDAtMS4zNDMtMS4zNDJMNiA0LjY1OCAyLjYwNCAxLjI2MmEuOTUuOTUgMCAxIDAtMS4zNDMgMS4zNDJMNC42NTcgNiAxLjI2MSA5LjM5NmEuOTUuOTUgMCAxIDAgMS4zNDMgMS4zNDJMNiA3LjM0MmwzLjM5NSAzLjM5NmEuOTQ2Ljk0NiAwIDAgMCAxLjM0My4wMDEuOTUuOTUgMCAwIDAgMC0xLjM0M0w3LjM0MiA2WiIgZmlsbD0iI2ZmZiIvPjwvZz48ZGVmcz48Y2xpcFBhdGggaWQ9ImEiPjxwYXRoIGZpbGw9IiNmZmYiIGQ9Ik0wIDBoMTJ2MTJIMHoiLz48L2NsaXBQYXRoPjwvZGVmcz48L3N2Zz4=");
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: "";
+  height: 12px;
+  margin-left: 10px;
+  margin-top: 7px;
+  min-width: 12px;
+  width: 12px
+}
+
+.prompt-blade {
+  align-items: center;
+  align-items: flex-start;
+  align-self: stretch;
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
+  border-radius: 10px;
+  cursor: pointer;
+  display: flow;
+  flex: 1 0 0;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px 20px 24px;
+  transition-delay: 3s;
+  transition-property: border;
+  max-width: 276px;
+}
+
+.prompt-blade:hover {
+  box-shadow: 3px 6px 6px 0 rgba(0, 0, 0, .16)
+}
+
+.prompt-blade:active {
+  border: 1px solid #095aba;
+  transition-delay: 0s
+}
+
+.prompt-icon {
+  margin-inline-end: 5px;
+  position: relative;
+  top: 5px
```

Comment:
This is a bit dangerous, since it takes the images outside of its regular flow. Why not handle it from the parent:
```
.prompt-prefix {
  display: flex;
  align-items: center;
}
```

and then you could remove these lines.